module/actor/actor.js

```diff
@@ -79,17 +84,25 @@ export class PrimePCActor extends Actor
 
 	async _prepareCharacterDataV2(data, actorData)
```

Comment:
Although now I look at it more closely I see that ultimately:
`actorSystemData` == `this.system`
and
`actorData` == `this`

Is there any point passing both in, wouldn't everything we need be available through `this`?

Reply:
In a word, no? In a few more words, probably not and it's probably a good idea to refactor it.

This has come about from originally copying the Foundry patterns and then upgrading/refactoring for v10. It's now a little on nonsensical side and needs reworking. The only thing I'm not 100% sure about is if:
`actorSystemData === this.system`

In other words is it a clone of the data or the actual data itself. I'm suspicious that _some_ of the fuckery I've been experiencing with getting the upgrade code to work is down to editing the wrong copy of things at the wrong times. But I've not been sure enough to deep dive into it whilst other lower hanging possibilities exist.

Reply:
Yeah - I did wonder about that myself, but it's just using a straight up `=` so it should just be a reference to `this.system` rather than a clone. I think 😅

Reply:
Yeah, I'm just not sure if the param `actorData` is properly equal to `this`.

Reply:
OK, well let's not rock the boat then, it was an aesthetic thing really.